keanu-python/tests/keanu_assert.py

```diff
@@ -0,0 +1,44 @@
+"""
+Some utility functions to compare tensor-like things
+They might be Keanu tensors or Numpy arrays
+"""
+def tensors_equal(t1, t2):
+    if not shapes_equal(__get_shape(t1), __get_shape(t2)):
+        return False
+        print("Shapes don't match", __get_shape(t1), __get_shape(t2))
+        return False
+    it1 = __as_list(t1)
+    it2 = __as_list(t2)
+    for i in range(len(it1)):
```

Comment:
perhaps a bit more pythonic to use:

for idx, value = enumerate(zip(t1, t2)) ?

Reply:
nice!